src/sdk-reference/cpp/1/kuzzle/add-listener/index.md

```diff
@@ -44,17 +45,8 @@ ERROR
 
 ### **listener**
 
-An instance of a class that inherits from `kuzzleio::EventListener`.  
-This class must implement the following method:
-
-```cpp
-class MyListener : public kuzzleio::EventListener {
-  public:
-    void trigger(char* json_payload) const {
-      // Do something with the json payload
-    }
-};
-```
+A c++11 lambda which take a `const std::string`
+Internally `EventListener` is a typedef on `const std::function<void(const std::string)>`.
```

Comment:
Why not juste `EventListener`is defined as `const std::function<void(const std::string)>`

There is no need for internally as far as this is public API and the user needs to know how it defined